.github/workflows/ci.yml

```diff
@@ -228,19 +228,21 @@ jobs:
             !:trino-main,
             !:trino-tests,
             !:trino-raptor-legacy,
-            !:trino-accumulo,
+            !:trino-accumulo,!:trino-accumulo-iterators,
```

Comment:
is this related to airbase upgrade?
what about other changes in ci.yml?

Reply:
It is related, as this module does not have tests and surefire now breaks when one is running `mvn test` without any test defined. It is to make sure that we don't run into a case where for example profile in maven is incorrectly defined.

Extracted this to separate line at the bottom. However I am not sure how to add a comment to the ci.yaml file.

Reply:
That's error prone. If someone adds tests, they silently will be ignored by CI.

Reply:
As a workaround, i'd rather add a dummy test class to each of these modules.